virttest/env_process.py

```diff
@@ -1191,7 +1200,7 @@ def preprocess(test, params, env):
 
     # Preprocess all VMs and images
     if params.get("not_preprocess", "no") == "no":
-        process(test, params, env, preprocess_image, preprocess_vm)
+        process(test, params, env, preprocess_image, preprocess_vms)
 
```

Comment:
Is there any practical implication for using the older ``preprocess_vm`` functions still and why they are still public?

Reply:
The `env_process.preprocess_vm` is frequently used in tests to initialize extra vm (or to redefine it with dynamic params)

Reply:
Fair enough, I was just wondering. My approach towards multi-vm tests would be to always leave the pre- and post-processing to the `env_process` before and after a test and just make sure all vms are available within the test with a few small exceptions that rather rely on `VM.create` but if it is useful then I don't see why not.